db/src/main/kotlin/edu/artic/db/AppDataManager.kt

```diff
@@ -93,9 +93,15 @@ class AppDataManager @Inject constructor(
 
                         val tours = result.tours
                         if (tours?.isNotEmpty() == true) {
+                            appDatabase.tourDao.clear()
                             appDatabase.tourDao.addTours(tours)
                         }
 
+                        val exhibitionsCMS = result.exhibitions
+                        if (exhibitionsCMS?.isNotEmpty() == true) {
+                            appDatabase.exhibitionCMSDao.addCMSExhibitions(exhibitionsCMS)
```

Comment:
Do we wanna clear this out before adding? I think soo.

Reply:
Fixed.